scripts/hammerstone/ui/uiManager.lua

```diff
@@ -72,26 +73,68 @@ function uiManager:initManageElements(gameUI, manageButtonsUI, manageUI)
 	local menuButtonSize = manageButtonsUI.menuButtonSize
 	local menuButtonPadding = manageButtonsUI.menuButtonSize * manageButtonsUI.menuButtonPaddingRatio
 	local toolTipOffset = manageButtonsUI.toolTipOffset
+	local originalButtonsCount = manageButtonsUI.menuButtonCount
+
+	-- Re-usable function to close the element's tab
+	local function closeTab()
+		manageUI:hide()
+		menuButtonsView.hidden = true
+	end		
 
 	-- Capture the last button in the row, as we will place new buttons offset from it.
 	local lastButton = manageButtonsUI.menuButtonsByManageUIModeType[#manageButtonsUI.menuButtonsByManageUIModeType]
 
+	-- Setup keybindings to close the tab
+	local keyMap = {
```

Comment:
I'm fine with this version, but you might want to look into the key-binding manager that Hammerstone offers. I think it's a really more convenient way to do key bindings !

Reply:
After discussion, we don't want a custom binding